tokens/eth/0xA62D9d68674AaAc7Cb31E8c0515Dd3A47D897363.json

```diff
@@ -0,0 +1,34 @@
+{
+    "symbol": "MANA",
+    "address": "0xA62D9d68674AaAc7Cb31E8c0515Dd3A47D897363",
+    "decimals": 18,
+    "name": "Aave Interest bearing KNC",
```

Comment:
This name, and the symbol are somewhat misaligned

Reply:
Thanks @blurpesec, I will fixed this.